SECURITY.md

```diff
@@ -1,23 +1,11 @@
-# Security Issue Policy
-
-If you uncover a security issue with Firecracker, please write to us on
-<firecracker-security-disclosures@amazon.com>. Please encrypt sensitive
-information using the [PGP key](PGP-KEY.asc).
-
-Once the Firecracker [maintainers](MAINTAINERS.md) become aware (or are made
-aware) of a security issue, they will immediately assess it. Based on impact and
-complexity, they will determine an embargo period (if externally reported, the
-period will be agreed upon with the external party).
-
-During the embargo period, maintainers will prioritize developing a fix over
-other activities. Within this period, maintainers may also notify a limited
-number of trusted parties via a pre-disclosure list, providing them with
-technical information, a risk assessment, and early access to a fix.
-
-The external customers are included in this group based on the scale of their
-Firecracker usage in production. The pre-disclosure list may also contain
-significant external security contributors that can join the effort to fix the
-issue during the embargo period.
-
-At the end of the embargo period, maintainers will publicly release information
-about the security issue together with the Firecracker patches that mitigate it.
+## Reporting Security Issues
+
+We take all security reports seriously.
+When we receive such reports,
+we will investigate and subsequently address
+any potential vulnerabilities as quickly as possible.
+If you discover a potential security issue in this project,
+please notify AWS/Amazon Security via our
+[vulnerability reporting page](http://aws.amazon.com/security/vulnerability-reporting/)
+or directly via email to [AWS Security](mailto:aws-security@amazon.com).
+Please do *not* create a public GitHub issue in this project.
```

Comment:
Nit
```suggestion
*Do not create a public GitHub issue.*
```

Reply:
Thanks! I would prefer to make no changes to the original.